cypress/integration/pages/photoGalleryPage/tests.js

```diff
@@ -0,0 +1,63 @@
+import config from '../../../support/config/services';
+import appConfig from '../../../../src/server/utilities/serviceConfigs';
+
+// For testing important features that differ between services, e.g. Timestamps.
+// We recommend using inline conditional logic to limit tests to services which differ.
+export const testsThatAlwaysRun = ({ service, pageType }) => {
+  describe(`No testsToAlwaysRun to run for ${service} ${pageType}`, () => {});
+};
+
+// For testing features that may differ across services but share a common logic e.g. translated strings.
+export const testsThatFollowSmokeTestConfig = ({ service, pageType }) => {
+  describe(`testsThatFollowSmokeTestConfig to run for ${service} ${pageType}`, () => {
+    it('should render a H1, which displays the headline', () => {
+      cy.request(`${config[service].pageTypes[pageType].path}.json`).then(
+        ({ body }) => {
+          cy.get('h1').should('contain', body.promo.headlines.headline);
+        },
+      );
+    });
+
+    it('should render a timestamp', () => {
+      cy.request(
+        `${config[service].pageTypes.photoGalleryPage.path}.json`,
+      ).then(({ body }) => {
+        const { lastPublished, firstPublished } = body.metadata;
+        const timeDifferenceMinutes =
+          (lastPublished - firstPublished) / 1000 / 60;
+        const minutesTolerance = 1;
+        cy.get('time')
```

Comment:
Sorry pretty new to Cypress, which element does this retrieve, I found these two `time` elements on the PGL page:
![image](https://user-images.githubusercontent.com/9645462/72968357-34aaaa00-3dbb-11ea-86d0-014ee36f2ac8.png)

Reply:
So this `get('time)` retrieves all the time elements and the `.eq(0)` grabs the first `<time>` tag. I'm also trying to understand cypress as I haven't written many tests before this :).

Reply:
Thanks makes sense now